<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAlarmDialog</class>
 <widget class="QDialog" name="NewAlarmDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Alarm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc/resources.qrc">
    <normaloff>:/MultiAlarm/speaker.ico</normaloff>:/MultiAlarm/speaker.ico</iconset>
  </property>
  <property name="toolTip">
   <string>Create a new alarm.</string>
  </property>
  <property name="statusTip">
   <string>Create a new alarm.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_name">
       <property name="maximumSize">
        <size>
         <width>290</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_message">
       <property name="maximumSize">
        <size>
         <width>290</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Alarm Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="m_timerRadio">
          <property name="text">
           <string>Timer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="m_timer">
          <property name="displayFormat">
           <string>HH:mm:ss</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="m_clockRadio">
          <property name="text">
           <string>Clock</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="m_clock">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="m_timerLoop">
        <property name="text">
         <string>Loop Timer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Progress Notification</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_colorComboBox">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Alarm sound</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_soundComboBox"/>
        </item>
        <item>
         <widget class="QToolButton" name="m_playSoundButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="rsc/resources.qrc">
            <normaloff>:/MultiAlarm/play.ico</normaloff>:/MultiAlarm/play.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="m_showTray">
        <property name="text">
         <string>Show progress in system tray.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_showDesktop">
        <property name="text">
         <string>Show progress in the desktop.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_buttons</sender>
   <signal>accepted()</signal>
   <receiver>NewAlarmDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttons</sender>
   <signal>rejected()</signal>
   <receiver>NewAlarmDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
